packages/react/src/components/cookieConsent/CookieConsent.stories.tsx

```diff
@@ -0,0 +1,459 @@
+import React, { useState } from 'react';
+
+import { SupportedLanguage } from './CookieConsentContext';
+import { ConsentsInModal } from './consentsInModal/ConsentsInModal';
+import { ConsentsInPage } from './consentsInPage/ConsentsInPage';
+import { getConsentStatus, hasHandledAllConsents } from './util';
+import { ContentSource, pickConsentIdsFromContentSource } from './content.builder';
+
+export default {
+  component: ConsentsInModal,
+  title: 'Components/CookieConsent',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const ModalVersion = (args) => {
+  const [language, setLanguage] = useState<SupportedLanguage>('fi');
+  const onLanguageChange = (newLang) => setLanguage(newLang);
+  const focusedElementAfterCloseId = 'focused-element-after-cookie-consent-closed';
+  const contentSource: ContentSource = {
+    siteName: 'Sivusto XXX',
```

Comment:
XXX-refers to old style adult pages, maybe we think some other name for the example? 😅

Reply:
:) Let's wait until we have real content